openam-core/src/main/java/com/iplanet/services/comm/server/PLLAuditor.java

```diff
@@ -86,7 +87,7 @@ public void auditAccessAttempt() {
 
             AuditEvent auditEvent = auditEventFactory.accessEvent(realm)
                     .forHttpServletRequest(httpServletRequest)
-                    .timestamp(startTime)
+                    .timestamp(startTime,auditEventPublisher.isLtzEnabled())
```

Comment:
This is a minute thing, but add a space after the comma please.

Reply:
Done.